README.md

```diff
@@ -270,6 +270,31 @@ If there's some error during compilation follow these steps:
 
 - Put here any general/platform tweaks that you need for _your_ specific environment. See `/local32|64/etc/profile2.local` for example usage.
 
+## Custom Patches
+
+--------
+
+- Using custom patches in not officially supported, if you do use custom patches, do not expect much support from this suite
+  - Either go to the patch's author or to a forum for help
+
+To use a custom patch, create a script within the build folder named `${REPO_NAME}_extra.sh` where `${REPO_NAME}` is something like "ffmpeg" or "x264", whatever the folder's name would be.
+
+Example Script: `/build/aom_extra.sh` for `aom-git`
+
+``` bash
+#!/bin/bash
+_pre_cmake(){
+    do_pacman_install libwebp
+    do_patch "https://gist.githubusercontent.com/1480c1/9fa9292afedadcea2b3a3e067e96dca2/raw/50a3ed39543d3cf21160f9ad38df45d9843d8dc5/0001-Example-patch-for-learning-purpose.patch" am
```

Comment:
Please, put another example using a local file.
Where targets the "relative" path if I use `do_patch "my-patch.diff" am` ?
And why the `am`?

Reply:
Sure, I guess I should specify the relative dir would be the cloned/untared folder and the am just means use git am for patch files( git apply but has commit information such as author)

Reply:
`am` uses `git am` to apply the patch, instead of `diff`.